src/providers/lambda.ts

```diff
@@ -221,13 +233,13 @@ export class LambdaRunner extends Construct implements IRunnerProvider {
   getStepFunctionTask(parameters: RunnerRuntimeParameters): stepfunctions.IChainable {
     return new stepfunctions_tasks.LambdaInvoke(
       this,
-      this.label,
+      this.labels.join(', '),
```

Comment:
Is the extra space here intentional?

Reply:
Yes. This is for display purposes and it's easier to read with the spaces.